app/models/pour.rb

```diff
@@ -45,10 +45,11 @@ def calculate_duration
   end
 
   def set_last_pour_at
-    return unless finished_at && (user_id_changed? || finished_at_changed?)
-    User.find(changes["user_id"] || [user_id]).each do |user|
-      user.last_pour_at = user.pours.finished.maximum(:created_at)
-      user.save
+    if finished_at && (user_id_changed? || finished_at_changed?)
```

Comment:
Huh, that's why that hasn't been working. This is in an after save, so `*_changed?` is always false

Reply:
ha!

Reply:
Also, technically, if the user_id changed, it should be updating 2 users. There are a couple of errors I've spotted on my home install that I should push updates for.